wiki/beginners.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Beginners
+parent: Wiki
+---
+
+This page was moved from the old wiki and is in the process of being revised.
```

Comment:
All these block comments appear a little off in the rendered output.

![Screenshot 2021-08-30 at 19-31-25 Beginners](https://user-images.githubusercontent.com/1645308/131380450-87a8e6f2-c1a1-4fff-89bd-d0bc31b33760.png)

Please consider shortening them. The reference to how things were before can be omitted, because any new visitor to the site will not need to know that there was an old wiki in the first place.

![Screenshot 2021-08-30 at 19-31-19 Beginners](https://user-images.githubusercontent.com/1645308/131380518-edbb1264-6d51-404b-9f16-30ff61c0a738.png)

This accounts for all of these blocks in the `wiki/` section.

Reply:
I think this can be discussed in another pull request.